sciencebox/values.yaml

```diff
@@ -587,6 +583,120 @@ swan:
     #       cert:
     #       key:
 
+## cbox web
+cernbox-web:
+  initContainers:
+    - name: install-web
+      image: busybox
+      command: ["/bin/sh", "-c"]
+      args: 
+        - wget -O /var/www/web/web-tar.gz https://github.com/cernbox/web-release/releases/download/v0.0.9/web.tar.gz;
```

Comment:
Template both version numbers (`v0.0.9` and `0.1.7`) since they appear in multiple places.

You can also `export` the path where you're going to de-archive the `tar.gz` to so you don't have extremely long lines on the initContainer script.